content/g-sgov/g-sgov-události/g-sgov-události-glosář-MaNe.ttl

```diff
@@ -56,7 +56,7 @@ g-sgov-události-pojem:partner-jako-fyzická-osoba
 g-sgov-události-pojem:má-hlavního-sponzora
         a       <http://www.w3.org/2004/02/skos/core#Concept> ;
         <http://www.w3.org/2004/02/skos/core#altLabel>
-                "Hlavní sponzor"@cs ;
+                "Hlavního sponzor"@cs ;
```

Comment:
tohle vypadá nečesky.

Reply:
trikrat jsem precetl Nečaský az jsem pochopil :)  

Kazdopadne, to byl dobry catch :) :+1: , diky .... ja jsem to revidoval, no spletl jsem si cervenu se zelenou barvou :)